scripts/drone/steps/lib.star

```diff
@@ -418,7 +418,7 @@ def test_a11y_frontend_step(edition, port=3001):
         },
         'failure': 'ignore',
         'commands': [
-            'yarn wait-on http://$HOST:$PORT',
+            'yarn wait-on http://$HOST:$PORT/login',
```

Comment:
Maybe we should have both lines? One waiting for the server and a second line for the actual page? Maybe each page needs to "warm-up" before we can run pa11y-ci?

Reply:
🤔 yes, maybe having both lines will give more time, I also found a property called `wait`, is a time on milliseconds that we can pass to each URL before Pa11y is executed.

Reply:
The `wait` prop seems promising 👍